templates/pages/about.html.twig

```diff
@@ -15,6 +15,11 @@
         developers and content-editors. Bolt is Open Source, and as such it uses other Open Source
         components. If you are a developer you're very welcome to help in the further development of Bolt.
     </p>
+    <p>The ongoing Bolt development is spearheaded by:</p>

```

Comment:
The ongoing Bolt development takes place under the care of:
- sounds better for me